### ISSUE TITLE
[Suggestion] Use GuiColorEditorHSV

### ISSUE THREAD

**Issue by Zailer43** [maruohon/malilib#74]
By creating a ConfigColor and clicking on the color rectangle to the right of the field one would expect to be able to choose the color in some more graphical way, but this is not the case, looking at the library code I saw that [GuiColorEditorHSV ](https://github.com/maruohon/malilib/blob/fabric_1.18.x/src/main/java/fi/dy/masa/malilib/gui/GuiColorEditorHSV.java)exists since 2019 and i saw that it is not used

![image](https://user-images.githubusercontent.com/63565983/154156326-21d108fc-bdbe-4b3b-bfab-a666a2129f1e.png)
![GuiColorEditorHSV](https://user-images.githubusercontent.com/63565983/154156514-16c44bc7-5427-4834-b5ed-62fe59dc08b0.png)

**Comment by maruohon** [comment]
It has been used in the 1.12.2 code base probably since late 2019, but all the 1.13+ versions are still based on 1.12.2 code from early June 2019, and the 1.12.2 code hasn't been merged to the 1.13+ versions since then. I think currently just MiniHUD uses the color editor screen for some shapes, but none of the config widgets in that old code hooked up the color editor screen to the color indicator widget yet.

So basically yes all the configs will be/are using it, just not in the 1.13+ code base yet. I'm currently still working on major malilib rewrites in 1.12.2, after which I'll be working on rewriting and cleaning up the other mods. There is a huge amount of stuff on my todo list still, so I can't give any kind of meaningful ETA for when the 1.12.2 code is ready to be ported forward again.
